src/components/ProgramsTable/ProgramsTable.jsx

```diff
@@ -61,4 +127,8 @@ class ProgramsTable extends Component {
   }
 }
 
+ProgramsTable.propTypes = {
+  programQueryData: PropTypes.shape({}).isRequired,
```

Comment:
Do we know the expected shape of the program query data? If we don't, we can leave it as `{}` for now. Otherwise, we should probably define the expected structure here.

Reply:
it's big and gross, so I left it generic. Also, while adding a test, i realized it's actually an array of shapes, so that's coming in a commit soon. This is what it might look like:

```
let programQueryData = [
    {
      node: {
        context: {
          programUUID: '6eefc008-db50-46f0-8746-667f55533a5d',
          programSlug: 'Example Program',
          programName: 'exampleprogram',
        },
      },
    },
    {
      node: {
        context: {
          programUUID: null,
          programSlug: null,
          programName: null,
        },
      },
    },
  ];
```